### DISCUSSION TITLE
Using MediaPlayerElement in WinUI template

### DISCUSSION THREAD

**Discussion by jp-weber** [unoplatform/uno#10980] [Q&A] [answered]
It is possible to use the MediaPlayerElement and other media player controls in the WinUI template? 

Currently I don't focus on an application with WinUI/Windows App Sdk, but on iOS and Android I would like to use the MediaPlayerElement of Uno.UI.
https://github.com/unoplatform/uno/tree/master/src/Uno.UI/UI/Xaml/Controls/MediaPlayer

**Comment by jeromelaban** [comment, marked as answer]
Thanks for the report. We're tracking this particular issue in https://github.com/unoplatform/uno/issues/10938. That control should definitely be available.
